src/tests/addTask.test.js

```diff
@@ -0,0 +1,15 @@
+const addTask = require('./_mock/addTask.js');
+
+describe('Adding new tasks', () => {
+  test('addTask function exists', () => {
+    expect(typeof addTask).toEqual('function');
+  });
+
+  test('should add a task to the tasks array', () => {
+    const tasks = [];
+    const description = 'Test task';
+    addTask(description);
+
+    expect(tasks).toHaveLength(0);
+  });
```

Comment:
- [ ] After adding a task to an empty task array the length of the array should be 1 instead of 0.

Reply:
Ok. Thanks.
changes done.